examples/trials/efficientnet/README.md

```diff
@@ -0,0 +1,24 @@
+# EfficientNet
+
+[arXiv](https://arxiv.org/abs/1905.11946)
+
+We provide here:
+
+* Search space and tuners for finding the best tuple (alpha, beta, gamma) for EfficientNet-B1 with grid search, as discussed in Section 3.3 in [paper](https://arxiv.org/abs/1905.11946).
+* Search space for EfficientNet transfer learning on datasets other than ImageNet; settings are borrowed from Appendix A.5 from "[Do Better ImageNet Models Transfer Better?](https://arxiv.org/abs/1805.08974)".
+
+## Instructions
+
+1. Clone our modified version of [EfficientNet-PyTorch](https://github.com/ultmaster/EfficientNet-PyTorch). The modifications were done to adhere to the original [Tensorflow version](https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet) as close as possible (including EMA, label smoothing and etc.); also added are the part which gets parameters from tuner and reports intermediate/final results. Clone it into `EfficientNet-PyTorch`; then you will see files like `main.py`, `train_imagenet.sh`, as specified in the configuration files.
+2. Resolve all the dependencies following the README.
+3. Run `nnictl create --config config_lr.yml` if you want to do transfer learning; or `nnictl create --config config_net.yml` if you want to find the best EfficientNet-B1. Adjust the training service (PAI/local/remote), batch size in the config files according to your environment.
+
+For training on ImageNet, you should read `EfficientNet-PyTorch/train_imagenet.sh`. Download ImageNet beforehand and extract it adhering to [PyTorch format](https://pytorch.org/docs/stable/torchvision/datasets.html#imagenet) and then replace `/mnt/data/imagenet` in with the location of your ImageNet storage. This file should also be a good example in case you want to mount ImageNet into the container on OpenPAI.
```

Comment:
generally speaking, you should provide several lines of simple instructions so that users can just copy your code and run :)